spec/features/circle_check_score_update_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe 'updating a circle check score' do 
+
+  before :each do 
+    create :circle_check_score
+  end
+
+
+  context 'with admin privilege' do 
+    it 'updates circle check score' do 
+      when_current_user_is :admin
+      visit circle_check_scores_url
+      fill_in 'circle_check_score_defects_found', with: '4'
+      click_on 'Save score'
+      expect(page).to have_text 'Score was saved.'
+      input = find_field :circle_check_score_defects_found
+      expect(input.value).to eql '4'
```

Comment:
On line 6 the CCS creates a score for us that we will update in the test with a different value. Which is the 4, it is filled in on line 14 so we test for a successful or unsuccessful update in the it tests.